lib/matplotlib/offsetbox.py

```diff
@@ -1500,16 +1500,23 @@ def __init__(self, ref_artist, use_blit=False):
             ref_artist.set_picker(True)
         self.got_artist = False
         self._use_blit = use_blit and self.canvas.supports_blit
-        self.cids = [
-            self.canvas.callbacks._connect_picklable(
-                'pick_event', self.on_pick),
-            self.canvas.callbacks._connect_picklable(
-                'button_release_event', self.on_release),
+        callbacks = ref_artist.figure._canvas_callbacks
+        self._disconnectors = [
```

Comment:
Do we need to drop this in `__getstate__`?

Reply:
Why would we?

Reply:
The `canvas` property has a note "property not attribute to keep pickable" which I understood to be because pickling the canvas was not always safe and this includes a reference to the canvas.

Reply:
There's no reference to the canvas, only to the figure?

Reply:
🤦🏻 I can not read (I was reading `ref_artist.figure.canvas.callbacks`) and forget we moved the callback registry to live on the figure.